rtgui/batchqueuepanel.h

```diff
@@ -60,22 +64,23 @@ class BatchQueuePanel : public Gtk::VBox,
     void init (RTWindow* parent);
 
     void addBatchQueueJobs(const std::vector<BatchQueueEntry*>& entries , bool head = false);
+    void saveOptions ();
+
+    bool handleShortcutKey (GdkEventKey* event);
```

Comment:
There must be a reason to make those accessible, but I can't see it from the diff.

Anyway, as one last nitpicking: Could you adapt the function order in the implementation to the one given in the header?

Reply:
There's only these two usages, I think:
https://github.com/Beep6581/RawTherapee/blob/b37d88fb2245fc99c529a377c57f3475c9da3c54/rtgui/rtwindow.cc#L674-L676
https://github.com/Beep6581/RawTherapee/blob/b37d88fb2245fc99c529a377c57f3475c9da3c54/rtgui/rtwindow.cc#L614-L616